scripts/build.sh

```diff
@@ -651,6 +651,9 @@ case "$product-$platform-$method" in
     RunXcodebuild \
       -scheme $product \
       "${xcb_flags[@]}" \
+      -test-iterations 10000 \
+      -retry-tests-on-failure \
+      -test-repetition-relaunch-enabled YES \
```

Comment:
Tried to repro the flaky test with Xcode 13's new test iteration flags. In the context of this PR, `FIRMessagingTokenManagerTest` ran 10000 times and I wasn't able to see any crashes.

Reply:
Could the issue be related to running multiple incarnations of the tests in parallel?

Reply:
Yeah, good point. I added back all of Messaging's tests and have only seen one flake so far. 4a11772 ran 10000 times of the Messaging spm tests

Reply:
Maybe try running all the platforms at the same time

Reply:
cc @chliangGoogle

Reply:
> Maybe try running all the platforms at the same time

How would this done? @paulb777